docs/source/eFeatures.rst

```diff
@@ -2000,6 +2000,37 @@ The burst detection can be fine-tuned by changing the setting strict_burst_facto
 
     numpy.array([spikes_per_burst[0] - spikes_per_burst[-1]])
 
+`Python efeature`_ : impedance
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Computes the impedance given a ZAP current input and its voltage response.
+It will return the frequency at which the impedance is maximal, in the range (0, impedance_max_freq] Hz,
+with impedance_max_freq being a setting with 50.0 as a default value.
+
+- **Required features**: current, LibV1:Spikecount, LibV5:voltage_base, LibV5:current_base
+- **Units**: Hz
+- **Pseudocode**: ::
+
+    normalized_voltage = voltage_trace - voltage_base
+    normalized_current = current_trace - current_base
+    if spike_count < 1:  # if there is no spikes in ZAP
+        fft_volt = numpy.fft.fft(normalized_voltage)
+        fft_cur = numpy.fft.fft(normalized_current)
+        # convert dt from ms to s to have freq in Hz
+        freq = numpy.fft.fftfreq(len(normalized_voltage), d=dt / 1000.)
+        if any(fft_cur) == 0:
+            return None
```

Comment:
My curiosity: can we not perform this check before computing the `fftfreq`? If yes, maybe we can avoid computing the `fftfreq when the condition holds true.

Reply:
thanks! done in latest commit